### PR TITLE
{phys}[gompi/2022a] LISFLOOD-FP v8.1 w/wo CUDA v11.7.0

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/l/LISFLOOD-FP/LISFLOOD-FP-8.1-gompi-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+dependencies = [
+    ('netCDF', '4.9.0')
+]
+
+_lisflood_bins = ['DG2downscale', 'generateDG2DEM', 'generateDG2start', 'lisflood']
+
+# It doesn't have a working make install, emulate in post-install step
+skipsteps = ['install']
+postinstallcmds = ["mkdir -p %(installdir)s/bin"]
+postinstallcmds += ["cp %s %%(installdir)s/bin/" % x for x in _lisflood_bins]
```
Comment (jfgrimm) [inline easybuild/easyconfigs/l/LISFLOOD-FP/LISFLOOD-FP-8.1-gompi-2022a.eb:45]: perhaps just use the `CMakeMakeCp` instead?

File: easybuild/easyconfigs/l/LISFLOOD-FP/LISFLOOD-FP-8.1-gompi-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,54 @@
+easyblock = 'CMakeMake'
```
Comment (jfgrimm) [inline easybuild/easyconfigs/l/LISFLOOD-FP/LISFLOOD-FP-8.1-gompi-2022a.eb:1]:
```suggestion
easyblock = 'CMakeMakeCp'
```

File: easybuild/easyconfigs/l/LISFLOOD-FP/LISFLOOD-FP-8.1-gompi-2022a.eb (hunk 1 above)
Comment (lexming) [reply easybuild/easyconfigs/l/LISFLOOD-FP/LISFLOOD-FP-8.1-gompi-2022a.eb:45]: good point, thanks

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3169.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/951e8bab0dbeb9a05a40898f5a4da3b1 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3169.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/ff1633a18d8afc138c38ea96e913d655 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @lexming!

**Pull request merged**